### PR TITLE
DOCSP-43243 Clarify driver support for language versions

### PR DESCRIPTION
**Pull request by lindseymoore**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43243>
Staging - <https://deploy-preview-943--docs-node.netlify.app/compatibility/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

**Review by alexbevi — APPROVED** [review]
LGTM

File: source/compatibility.txt [hunk 1]
```diff
@@ -33,13 +33,23 @@ The first column lists the driver version.
...
+The following compatibility table specifies the supported versions of
 the {+driver-long+} for use with a specific version of Node.js.
 
 The first column lists the driver version.
 
 .. include:: /includes/language-compatibility-table-node.rst
 
+.. note:: The {+driver-short+} Supports Node.js Versions in the "Active LTS" Phase or Later
+
+   After release, Node.js versions have a "Current" release status for six months
```
Comment (mcmorisi) [inline source/compatibility.txt:46]: Q: does "Current" need to be capitalized?

File: source/compatibility.txt [hunk 2]
```diff
@@ -33,13 +33,23 @@ The first column lists the driver version.
...
 The first column lists the driver version.
 
 .. include:: /includes/language-compatibility-table-node.rst
 
+.. note:: The {+driver-short+} Supports Node.js Versions in the "Active LTS" Phase or Later
+
+   After release, Node.js versions have a "Current" release status for six months 
+   to give library authors time to add support for them. The {+driver-long+} 
+   officially supports Node.js versions once their release status changes 
+   from "Current" to "Active LTS". To learn more about release statues for
```
Comment (mcmorisi) [inline source/compatibility.txt:49]: [Style guide on quotation marks](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#quotation-marks) says that we shouldn't use them after introducing the terms in question (current and active lts).

Comment (mcmorisi) [inline source/compatibility.txt:49]:
```suggestion
   from "Current" to "Active LTS". To learn more about release statuses for 
```

**Review by mcmorisi — APPROVED** [review]
LGTM with some tweaks!

File: source/compatibility.txt (hunk 1 above)
Comment (lindseymoore) [reply source/compatibility.txt:46]: Capitalized it, as it is in the Node.js docs.

**Pull request merged**